### PR TITLE
Add filter to use ath9k_htc for mesh point by default

### PR DESCRIPTION
**Pull request by benhylau**
I have this for my custom configs so the tp-link 722n is brought up as mp by default. Not sure if you want this as default.

### CODE REVIEW

**Comment by benhylau** [comment]
This also requires we switch to `freq=2412 HT40+`, thoughts?

File: packages.d/_ALWAYS.customise.add/etc/systemd/network/85-wlan-mesh.link [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # By default, all wlan interfaces will be access points
 [Match]
 Type=wlan
-Driver=rt2800usb
+Driver=ath9k_htc rt2800usb
```
Comment (hamishcoleman) [inline packages.d/_ALWAYS.customise.add/etc/systemd/network/85-wlan-mesh.link:5]: This change makes the below comment unclear.

Perhaps this change is a trigger to move this file out of the generic builder?

**Comment by benhylau** [comment]
I see the only reason to keep this in debian-minimal-builder is to show an example. Closing this and putting another version of this in mesh-orange: https://github.com/tomeshnet/mesh-orange/pull/25

**Comment by hamishcoleman** [comment]
At some point, I will probably try to pare this down to a more clear example.

**Pull request closed without merging**